FunctorCategories/gap/precompiled_categories/FinBouquetsPrecompiled.gi

```diff
@@ -225,7 +227,7 @@ function ( cat_1, source_1, range_1, alpha_1 )
     deduped_60_1 := deduped_64_1[3];
     deduped_59_1 := deduped_64_1[2];
     deduped_58_1 := CreateCapCategoryObjectWithAttributes( deduped_62_1, Length, 2 );
```

Comment:
Why isn't this `BigInt( 2 )`?

Reply:
and at several other places as well.

Reply:
See my comment above: https://github.com/homalg-project/CategoricalTowers/pull/185#issuecomment-1516664502